src/tool_getparam.c

```diff
@@ -813,7 +813,7 @@ ParameterError getparameter(const char *flag, /* f or -long-flag */
         break;
       case 'x': /* --krb */
         /* kerberos level string */
-        if(curlinfo->features & CURL_VERSION_KERBEROS4)
+        if(curlinfo->features & CURL_VERSION_SPNEGO)
```

Comment:
Why isn't that `CURL_VERSION_KERBEROS4 | CURL_VERSION_SPNEGO` as in the commit you linked?

Reply:
Per commit 569288b3bf11d364361ebb3b2d1eb19fda628704 support for Kerberos V4 was removed in `curl-7.33.0`.  I thought we kept the define in `curl.h` just for backward compatibility.  `CURL_VERSION_KERBEROS4` does not seem to be used anywhere else in the current code.